src/Altinn.AccessManagement.Core/Configuration/AppSettings.cs

```diff
@@ -0,0 +1,211 @@
+using Altinn.AccessManagement.Core.Models;
+using Newtonsoft.Json;
```

Comment:
I understand this is reuse, but 99% is not needed in access management. It is maybe more correct to have platform settings?